testing/pkg/tests/preconf/preconf.go

```diff
@@ -569,3 +579,30 @@ func getRevertingTxns(
 
 	return revertingTxns, nil
 }
+
+const (
+	PRECISION           = 1e16
+	ONE_HUNDRED_PERCENT = 100 * PRECISION
+)
+
+func computeResidualAfterDecay(startTimestamp, endTimestamp, commitTimestamp uint64) *big.Int {
+	if startTimestamp >= endTimestamp || startTimestamp > commitTimestamp || endTimestamp <= commitTimestamp {
+		return big.NewInt(0)
+	}
+
+	// Calculate the total time in seconds
```

Comment:
These comments don't add much value, if any, so rewrite the code rather than putting in so many comments.